public/app/app.bootstrap.js

```diff
@@ -80,6 +86,7 @@
     }).then(function(results) {
       collabosphere.constant('me', results.me.data);
       collabosphere.constant('config', results.config.data);
+      collabosphere.constant('deepLinkId', parameters.deep_link_id);
```

Comment:
Not that I know of, @johncrossman; if we're hitting this block of code every time and it's explicitly set to null when missing, I think we're good.